x-pack/plugins/fleet/server/services/epm/elasticsearch/transform/install.ts

```diff
@@ -115,17 +116,27 @@ async function handleTransformInstall({
   callCluster: CallESAsCurrentUser;
   transform: TransformInstallation;
 }): Promise<EsAssetReference> {
-  // defer validation on put if the source index is not available
-  await callCluster('transport.request', {
-    method: 'PUT',
-    path: `/_transform/${transform.installationName}`,
-    query: 'defer_validation=true',
-    body: transform.content,
-  });
-
+  try {
+    // defer validation on put if the source index is not available
+    await callCluster('transport.request', {
+      method: 'PUT',
+      path: `/_transform/${transform.installationName}`,
+      query: 'defer_validation=true',
+      body: transform.content,
+    });
+  } catch (err) {
+    // swallow the error if the transform already exists.
```

Comment:
LGTM, but I'd like to double-check with @kevinlog that this is safe.

Reply:
this looks okay with me - I will get @pzl to comment as well asap

Reply:
putting the same version, yeah we should logically be able to assume (aside from local dev situations) that it contains the same content trying to be installed then. Looks good.

Generally, the transform is _deleted_ before install, right? I wonder why that isn't happening here

Reply:
The transform is deleted if the saved object contains the transform, if our saved object and the state of ES differ this could happen